mbed_host_tests/host_tests_runner/mbed_base.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, options):
         self.serial_baud = DEFAULT_BAUD_RATE
         self.serial_timeout = 1
 
+        self.tags = self.options.tag_filters
```

Comment:
Please drop this assignment, and the subsequent line. You are not using `self.tags` anywhere.

Reply:
Do we need the other self.option assignments above this as well? 
`self.retry_copy = self.options.retry_copy`, etc are part of the same argument flow. I can run without this line since we define the value initially when we create the [config dictionary](https://github.com/kegilbert/htrun/blob/ffbcba2d104bea6d5167e020897b893fb37d3e87/mbed_host_tests/host_tests_runner/host_test_default.py#L159), should I clean up the other assignments or just leave this line out?

Reply:
> should I clean up the other assignments or just leave this line out?

Yes, but could you keep it in a separate commit. That way it's easier to see that it's an unrelated refactor.